service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RespondentController.java

```diff
@@ -109,6 +110,13 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
             .findAny()
             .ifPresent(date -> errors.add("Date of birth cannot be in the future"));
 
+        caseData.getAllRespondents().stream()
+            .map(Element::getValue).filter(respondent -> YES.getValue().equals(respondent.getLegalRepresentation()))
+            .map(Respondent::getSolicitor)
+            .filter(solicitor -> (solicitor.getLastName() == null))
+            .findAny()
+            .ifPresent(solicitor -> errors.add("Enter a representative name"));
+
```

Comment:
Doing this as I think we need to ensure a last name has been entered for the representative or we could end up with a null value when send the email out

Reply:
I agree, did you flag this with business? good to confirm with them that we are making representative name mandatory for submission

Reply:
Looks like this might be handled slightly differently so should ignore this validation for now while I wait to hear back from business